source/whats-new.txt

```diff
@@ -44,6 +45,40 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-6.2:
+
+What's New in 6.2
+-----------------
+
+The {+driver-short+} v6.2 release includes the following features:
+
+- Updates the ``bson`` package version to 6.2.0. This version includes
+  color visualization of types, as shown in the following image:
+
+  .. figure:: /includes/figures/bson-color.png
+     :alt: A screenshot of the terminal that shows printing in color
```

Comment:
Nice example image!

Suggestion:
No change necessary, but if it doesn't take much time, I think it could conform better if it used the Source Code Pro font (LG design specification for monospace text).

Reply:
changed the image